biggraphite/drivers/memory.py

```diff
@@ -118,14 +116,26 @@ def create_metric(self, metric):
 
     @staticmethod
     def __glob_names(names, glob):
-        dots_count = glob.count(".")
-        glob_re = re.compile(fnmatch.translate(glob))
-
-        return [
-            name for name in names
-            if name.count(".") == dots_count and glob_re.match(name)
+        dots = glob.count(".")
+        literal_glob_components = [
+            (index, component)
+            for (index, component) in enumerate(glob.split("."))
+            if not bg_glob._is_graphite_glob(component)
```

Comment:
Can we avoid *graphite* specific thing in the drivers ?
In this case:
* basic wildcard globbing should be done in the driver
* advanced integration specific globbing (graphite) should be callbed by the driver

Suggestion: add a specific method for wildcard globbing in bg_glob that could be callbed by drivers who can't do wildcards themselves